spec/controllers/news_tematica/news_tematicas_controller_spec.rb

```diff
@@ -1,12 +1,11 @@
-# coding: utf-8
-
 require 'spec_helper'
 
-describe NewsTematica::NewsTematicasController do
+describe NewsTematica::NewsTematicasController, type: :controller do
   render_views  # Necesario para que funcione el render_to_string usado para generar el html
   routes { NewsTematica::Engine.routes }
 
-  let!(:mi_news_tematica) { FactoryGirl.create(:news_tematica, tematica: Tematica.find_by_nombre('Bolsa'), fecha_desde: 7.days.ago, fecha_hasta: 1.minute.ago) }
+  let!(:mi_news_tematica) { FactoryGirl.create(:news_tematica, tematica: Tematica.find_by(nombre: 'Bolsa'),
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.